service/raft/raft_group0.hh

```diff
@@ -103,6 +104,8 @@ class raft_group0 {
     raft_group0_client& _client;
     seastar::scheduling_group _sg;
 
+    shared_ptr<group0_state_id_handler> _state_id_handler;
```

Comment:
Why is it shared and not unique? Or even just an embedded? Well, may be not embedded because `_gossiper` is unavailable at the creation time.